criu/net.c

```diff
@@ -2059,6 +2059,7 @@ static inline int restore_iptables(int pid)
 		return -1;
 	if (empty_image(img)) {
 		ret = 0;
+		close_image(img);
 		goto ipt6;
```

Comment:
There is one more resource leak in `criu/net.c`, `tmp_file` is not closed in case of error in the `restore_ip_dump()` function. @adrianreber Could you fix this one as well, please?

Reply:
done, please have a look

Reply:
thank you!